demos/tests/cases.py

```diff
@@ -128,7 +128,7 @@ def single_option_cases(key, *args):
     )),
 
     CppDemo(name='gaze_estimation_demo',
-            model_keys=['-m', '-m_fd', '-m_hp', '-m_lm'],
```

Comment:
@legendary111 please rebase your branch to actual develop, to not introduce unrelevant changes